mms/metrics/push_example.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Add another folder here for tools or examples or some such .. I don't think example code should be a part of the main source code.

Reply:
agreed will move it to a examples folder